lib/glific/third_party/bigquery/bigquery_schema.ex

```diff
@@ -2012,6 +2012,51 @@ defmodule Glific.BigQuery.Schema do
     ]
   end
 
+  @doc """
+  Schema for the contact_groups table
+  """
+  @spec contacts_groups_schema :: list()
+  def contacts_groups_schema do
+    [
+      %{
+        description: "Unique ID for the contact groups",
+        name: "id",
+        type: "INTEGER",
+        mode: "REQUIRED"
+      },
+      %{
+        description: "Reference for the contact",
+        name: "contact_id",
+        type: "INTEGER",
+        mode: "NULLABLE"
+      },
+      %{
+        description: "Reference for the collection",
+        name: "group_id",
+        type: "INTEGER",
+        mode: "NULLABLE"
```

Comment:
we need contact name, phone number and group label as well so people don't need to merge different table to get basic info